src/holofuel/components/layout/PrimaryLayout/PrimaryLayout.js

```diff
@@ -17,11 +18,14 @@ export function PrimaryLayout ({
   children,
   headerProps = {}
 }) {
-  const { data: { holofuelActionableTransactions: transactions = [] } = {} } = useQuery(HolofuelActionableTransactionsQuery, { fetchPolicy: 'network-only' })
+  const { data: { holofuelActionableTransactions: actionableTransactions = [] } = {} } = useQuery(HolofuelActionableTransactionsQuery, { fetchPolicy: 'network-only' })
   const { loading: holofuelUserLoading, data: { holofuelUser = {} } = {} } = useQuery(HolofuelUserQuery)
   const { data: { holofuelLedger: { balance: holofuelBalance } = { balance: 0 } } = {} } = useQuery(HolofuelLedgerQuery, { fetchPolicy: 'network-only' })
 
-  const inboxCount = transactions.length
+  const inboxCount = actionableTransactions.filter(actionableTx => (actionableTx.status !== STATUS.canceled) && (actionableTx.type !== TYPE.request)).length
```

Comment:
I made similar comments on `Inbox`, and looking here it's even clearer to me. We should be handling this with different graphql queries. There should at least be two different queries, one that returns *actually* actionable transactions (ie not cancelled), and a different one that returns *all* actionable transactions.

Reply:
Agree and decided on this approach (only for now) to avoid adding additional heavy calls to the page.  Refer to previous comment in Inbox.

Reply:
Agreed, and left comments as to why this is implemented in this case (for now).  See comment above. :+1: